chef_master/source/style_guide.rst

```diff
@@ -91,23 +91,35 @@ Bulleted Lists
 -----------------------------------------------------
 Bulleted lists break up text blocks and draw attention to a group of items::
 
-   * text goes here
-   * text goes here
-   * text goes here
-   * text goes here
+  Introductory sentence:
 
-Use the asterisk symbol (*) only for bulleted lists, even though Sphinx supports using other symbols.
+    * list item
+    * list item
+    
+      * sublist item
+      * sublist item
+      
+    * list item
+    * list item
+
+Use the asterisk symbol (*) only for bulleted lists, even though Sphinx supports using other symbols. Leave a blank line before the first list item and after the last, and leave blank lines around nested list items.
 
 Numbered Lists
 -----------------------------------------------------
 Numbered lists are created like this::
 
-   #. text goes here
-   #. text goes here
-   #. text goes here
-   #. text goes here
+  Introductory sentence:
+
+    #. numbered list item
+    #. numbered list item
+
+        #. sublist item 1
+        #. sublist item 2
+        #. sublist item 3
+
+    #. numbered list item
 
-Use the number symbol (#) to let Sphinx handle the actual ordering. If the number list needs to change later, you don't have to worry about making sure the numbers are in the correct order.
+Use the number symbol (#) to let Sphinx handle the actual ordering. If the number list needs to change later, you don't have to worry about making sure the numbers are in the correct order. Leave a blank line before the first list item and after the last, and leave blank lines around nested list items.
```

Comment:
Do you need to include a note about how many spaces are between each list level (+4)?